io/plugins/eu.esdihumboldt.hale.io.xls/src/eu/esdihumboldt/hale/io/xls/writer/XLSInstanceWriter.java

```diff
@@ -170,14 +174,18 @@ private void addSheetByQName(boolean solveNestedProperties, InstanceCollection i
 			try {
 				instance = instanceIterator.next();
 			} catch (NoSuchElementException e) {
+				if (!ignoreEmptyFeaturetypes) {
+					Sheet sheet = workbook.createSheet(selectedTypeName.getDisplayName());
+					Row headerRow = sheet.createRow(0);
+					writeHeaderRow(headerRow, headerRowStrings);
```

Comment:
Where is `headerRowStrings` populated for this type?
To me it looks like it will only contain something if a previous sheet contained instances and then it will be the header of that sheet repeated and not a header based on the type for this sheet.

Reply:
Very true indeed. 
I'll dig into it further to see how can I get the properties of a TypeDefinition without having an instance associated.